### PR TITLE
Implemented Google forms data import

### PR DESCRIPTION
**Pull request by BNazar00**
## Checklist:

-   [x] I have added tests with code coverage `>=75%`
-   [x] My code passes Checkstyle rules
-   [x] PR is reviewed manually again (to make sure you have 100% ready code)
-   [ ] All reviewers agreed to merge the PR

### CODE REVIEW

File: src/main/java/com/softserve/teachua/service/impl/CertificateExcelServiceImpl.java [hunk 1]
```diff
@@ -191,101 +201,137 @@ private void validateCertificateExcel(CertificateExcel certificateExcel, int row
...
-        if (!value.matches(DATE_FORMAT_REGEX)) {
-            resultList.add(new String[] {DATE_ERROR + messageDescription, "1"});
+            for (int i = 0; i < columnNames.size(); i++) {
+                sheet.autoSizeColumn(i);
+                sheet.setColumnWidth(i, Math.min(sheet.getColumnWidth(i), 100 * 256));
+            }
+
+            workbook.write(outputStream);
+        } catch (IOException e) {
+            log.error("Error creating .excel file of bad certificates values");
```
Comment (Vekeryk) [inline src/main/java/com/softserve/teachua/service/impl/CertificateExcelServiceImpl.java:332]: Should we just log and return outputStream.toByteArray(); on this exception?

**Comment by Vekeryk** [comment]
This refactoring using Message enum and pairs is pretty good

File: src/main/java/com/softserve/teachua/service/impl/CertificateExcelServiceImpl.java (hunk 1 above)
Comment (BNazar002) [reply src/main/java/com/softserve/teachua/service/impl/CertificateExcelServiceImpl.java:332]: I'd rather not do that, logging a whole byte array isn't so necessary in this case. Although, untill that moment, even if we catch an exception, we will have logged all the necessary data to debug this method.

Comment (BNazar00) [reply src/main/java/com/softserve/teachua/service/impl/CertificateExcelServiceImpl.java:332]: I see, throw an exception would be a good move

**Review by softservedata — APPROVED** [review]

**Pull request merged**